src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -0,0 +1,88 @@
+import { clone, reduce, keys } from 'underscore';
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import StateModel from './schedule_state';
+
+import FiltersApp from './filters_app';
+
+import { LayoutView, ScheduleTitleView, TableHeaderView } from 'js/views/patients/schedule/schedule_views';
+
+export default App.extend({
+  StateModel,
+  childApps: {
+    filters: {
+      AppClass: FiltersApp,
+      regionName: 'filters',
+      restartWithParent: false,
+      getOptions: ['currentClinician'],
+    },
+  },
+  stateEvents: {
+    'change:filters': 'restart',
+  },
+  onBeforeStart() {
+    if (this.isRestarting()) {
+      this.showScheduleTitle();
+      this.showTableHeaders();
+      this.getRegion('list').startPreloader();
+      return;
+    }
+
+    this.showView(new LayoutView({
+      state: this.getState(),
+    }));
+
+    this.getRegion('list').startPreloader();
+
+    this.showScheduleTitle();
+    this.startFiltersApp();
+    this.showTableHeaders();
+  },
+  beforeStart() {
+    const filter = this.getState().getEntityFilter();
+    return Radio.request('entities', 'fetch:actions:collection', { filter });
+  },
+  onStart(options, collection) {
+    this.collection = collection;
+  },
+  showScheduleTitle() {
+    const filters = this.getState().getFilters();
+    const owner = Radio.request('entities', 'clinicians:model', filters.clinicianId);
+    const role = Radio.request('entities', 'roles:model', filters.roleId);
+
+    this.showChildView('title', new ScheduleTitleView({
```

Comment:
we can probably pass the clinician as the model for this view, if we're not passing the state